frontend/src/metabase/dashboard/components/add-card-sidebar/QuestionPicker.jsx

```diff
@@ -23,8 +24,6 @@ import {
 import { SEARCH_DEBOUNCE_DURATION } from "metabase/lib/constants";
 import SelectList from "metabase/components/SelectList";
 
-const { isRegularCollection } = PLUGIN_COLLECTIONS;
```

Comment:
It's not safe to destructure mutable imports like this. Not related to the PR through.